<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDialog</class>
 <widget class="QDialog" name="GameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="guessButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Send guess</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="playerTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>10</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>26</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>26</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="guessTable">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>10</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>26</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>26</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your guess table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
